src/tests/api/test_api.py

```diff
@@ -3328,9 +3331,10 @@ def test_reset_metrics_ok(self):
     test_SubServiceIoTADevice.test_post_ok2()
     test_SubServiceIoTADevice.test_post_ok3()
 
-    test_SubServiceIoTAService = Test_SubServiceIoTAService_RestView()
-    test_SubServiceIoTAService.test_post_ok()
-    test_SubServiceIoTAService.test_post_ok2()
+    # RegisterService was used by legacy BlackButton scenario
```

Comment:
Leftover comments?

Reply:
Provision of blackbutton was handlec by ORC. Who knows...

Reply:
Ok, good point to keep it.

However, I'd suggest to add a comment explaining why this is commented, e.g.:

```
# Blackbutton test has been disabled given we are no longer supporting this bla bla
```